chia/server/server.py

```diff
@@ -345,7 +345,7 @@ async def start_client(
         session = None
         connection: Optional[WSChiaConnection] = None
         try:
-            timeout = ClientTimeout(total=30)
+            timeout = ClientTimeout(total=10)
```

Comment:
Not sure if we should change this, this doesn't affect things now that we have tasks, correct?